config/application.yml.default

```diff
@@ -337,7 +337,7 @@ development:
   identity_pki_local_dev: true
   in_person_proofing_enabled: true
   inherited_proofing_enabled: true
-  va_inherited_proofing_mock_enabled: true
+  va_inherited_proofing_mock_enabled: false
```

Comment:
If this override value is the same as the default, we could just remove it altogether.

```suggestion
```